components/Navbar.jsx

```diff
@@ -17,9 +19,17 @@ const StyledNavbar = styled.nav`
 `;
 
 export default function Navbar() {
+  const [showPlusIcon, setIcon] = useState(false);
+  const { pathname } = useRouter();
+  useEffect(() => {
+    setIcon(pathname === "/addChildPage");
+  }, [pathname, showPlusIcon]);
```

Comment:
I am fairly certain that we can get rid of both `useState` and `useEffect`, because `showPlusIcon` is derived directly from `pathname`:

```suggestion
  const { pathname } = useRouter();
  const showPlusIcon = pathname === "/addChildPage";
```